tool/kokoro/deploy.sh

```diff
@@ -4,7 +4,18 @@ source ./tool/kokoro/setup.sh
 setup
 
 echo "kokoro build start"
-./bin/plugin make --channel=dev
+
+./bin/plugin make --channel=dev -o4.0 -u -m1
+./bin/plugin make --channel=dev -o4.1 -u -m2
+
+curl https://download.java.net/java/GA/jdk11/9/GPL/openjdk-11.0.2_osx-x64_bin.tar.gz > ../java.tar.gz
```

Comment:
Is it worth moving this into a shared file? Or will it likely be unnecessary soon?

Reply:
I considered doing that. I decided not to because it will make undoing this change easier. But it is going to be more difficult to maintain as we have to edit not just the product matrix but also these files. So I'll probably refactor it the next time the product matrix changes.